src/ethereum_clis/clis/execution_specs.py

```diff
@@ -102,3 +110,141 @@ def is_fork_supported(self, fork: Fork) -> bool:
         `ethereum-spec-evm` appends newlines to forks in the help string.
         """
         return (fork.transition_tool_name() + "\n") in self.help_string
+
+
+class ExecutionSpecsExceptionMapper(ExceptionMapper):
+    """
+    Translate between EEST exceptions and error strings returned by execution specs t8n.
+    """
+
+    @property
+    def _mapping_data(self):
+        return [
+            ExceptionMessage(
+                TransactionException.TYPE_4_TX_CONTRACT_CREATION,
+                "Failed transaction: ",
+            ),
+            ExceptionMessage(
+                TransactionException.INSUFFICIENT_ACCOUNT_FUNDS,
+                "ailed transaction: ",
+            ),
+            ExceptionMessage(
+                TransactionException.TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED,
+                "iled transaction: ",
+            ),
+            ExceptionMessage(
+                TransactionException.INSUFFICIENT_MAX_FEE_PER_BLOB_GAS,
+                "led transaction: ",
+            ),
+            ExceptionMessage(
+                TransactionException.INSUFFICIENT_MAX_FEE_PER_GAS,
+                "ed transaction: ",
```

Comment:
These strings look very strange and deserve an explanation :slightly_smiling_face: 

I think this is because EELS doesn't yet have unique exceptions and you're just trying to ensure that the bidict has unique keys (in the reverse direction), right? And still correctly catch expected exceptions?

How about adding a ticket with a list of the missing exceptions to execution-specs for easy tracking and linking it here. I think the EELS Team still needs to know the list of exceptions they have to improve exceptions strings for, too?

Reply:
the issue is opened https://github.com/ethereum/execution-specs/issues/1021